### PR TITLE
Trio-analysis tutorial: Update PyEGA version & add creator name, license, and tests

### PR DESCRIPTION
**Pull request by JasperO98**

### CODE REVIEW

File: topics/variant-analysis/tutorials/trio-analysis/workflows/main_workflow.ga [hunk 1]
```diff
@@ -2,23 +2,18 @@
...
     "creator": [
-        {
-            "class": "Person",
-            "identifier": "",
-            "name": "Jasper Ouwerkerk"
-        },
         {
             "class": "Organization",
-            "name": "Erasmus Medical Center"
+            "name": "Jasper Ouwerkerk"
```
Comment (hexylena) [inline topics/variant-analysis/tutorials/trio-analysis/workflows/main_workflow.ga:7]: ah you are probably not an organisation, you're a person.

File: topics/variant-analysis/tutorials/trio-analysis/workflows/main_workflow.ga [hunk 2]
```diff
@@ -71,53 +67,55 @@
...
-            "tool_version": "4.0.0+galaxy0",
+            "tool_state": "{\"action\": {\"action_type\": \"list_dataset_files\", \"__current_case__\": 0, \"dataset_id\": \"EGAD00001008392\"}, \"output_log\": false, \"__page__\": null, \"__rerun_remap_job_id__\": null}",
+            "tool_version": "4.0.5+galaxy2",
             "type": "tool",
             "uuid": "265bf049-ac9f-4a59-bba3-311c6c818121",
+            "when": null,
             "workflow_outputs": [
                 {
-                    "label": "List of files in EGAD00001008392",
+                    "label": null,
```
Comment (wm75) [inline topics/variant-analysis/tutorials/trio-analysis/workflows/main_workflow.ga:88]: Please make sure that "workflow_outputs" for every step have labels.
The "best practice" wizard in the WF editor should also warn about this.

Comment (hexylena) [reply topics/variant-analysis/tutorials/trio-analysis/workflows/main_workflow.ga:88]: We don't current mandate that. Are the workflow output labels so important that the GTN should start mandating it?

Personally I find renaming every single output file a bit frustrating to do when they most have already ok names, but maybe that's just me being difficult. And especially in training workflows we often keep every output by default which planemo also says is bad but, isn't really as relevant for the training case.

File: topics/variant-analysis/tutorials/trio-analysis/workflows/main_workflow.ga [hunk 3]
```diff
@@ -3,7 +3,8 @@
     "annotation": "Trio Analysis using Synthetic Datasets from RD-Connect GPAP",
     "creator": [
         {
-            "class": "Organization",
+            "class": "Person",
+            "identifier": "",
```
Comment (hexylena) [inline topics/variant-analysis/tutorials/trio-analysis/workflows/main_workflow.ga:7]: you can fill your Orcid here if you like.

Comment (JasperO98) [reply topics/variant-analysis/tutorials/trio-analysis/workflows/main_workflow.ga]: Thanks!

File: topics/variant-analysis/tutorials/trio-analysis/workflows/main_workflow.ga (hunk 2 above)
Comment (hexylena) [reply topics/variant-analysis/tutorials/trio-analysis/workflows/main_workflow.ga]: @JasperO98 whenever you have time, if we could trouble you to select the important outputs from the workflow and label those, that would be helpful. We will begin mandating it.

Comment (hexylena) [reply topics/variant-analysis/tutorials/trio-analysis/workflows/main_workflow.ga]: @JasperO98 whenever you have time, if we could trouble you to select the important outputs from the workflow and label those, that would be helpful.

Comment (hexylena) [reply topics/variant-analysis/tutorials/trio-analysis/workflows/main_workflow.ga]: @JasperO98 whenever you have time, if we could trouble you to select the important outputs from the workflow and label those, that would be helpful. We will begin mandating it.

Comment (hexylena) [reply topics/variant-analysis/tutorials/trio-analysis/workflows/main_workflow.ga]: (sorry for the triple posting. gh was being unhelpful.)

Comment (JasperO98) [reply topics/variant-analysis/tutorials/trio-analysis/workflows/main_workflow.ga]: The workflow_output labels should be added already.

**Review by shiltemann — APPROVED** [review]

**Comment by shiltemann** [comment]
is this ready to go?

**Pull request merged**